earthaccess/search.py

```diff
@@ -1,16 +1,53 @@
 import datetime as dt
 from inspect import getmembers, ismethod
-from typing import Any, Dict, List, Optional, Tuple, Type, Union
+from typing import Any, List, Optional, Tuple, Type, Union
 
 import dateutil.parser as parser  # type: ignore
-from cmr import CollectionQuery, GranuleQuery  # type: ignore
-from requests import exceptions, session
+from cmr import CollectionQuery, GranuleQuery
+from requests import exceptions, get, session
 
+# type: ignore
 from .auth import Auth
 from .daac import find_provider, find_provider_by_shortname
 from .results import DataCollection, DataGranule
 
 
+def get_results(request, limit: int = 2000) -> list:  # type: ignore
+    """
+    Get all results up to some limit, even if spanning multiple pages.
+
+    :limit: The number of results to return
+    :returns: query results as a list
```

Comment:
I know our code so far is very inconsistent on this, but we're trying to standardize around [Google-style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) docstrings.

Reply:
Fixed in latest iteration. I _think_